reference/docs-conceptual/learn/deep-dives/everything-about-pscustomobject.md

```diff
@@ -41,6 +41,13 @@ You can then access and use the values like you would a normal object.
 $myObject.Name
 ```
 
+To get name or value by index-number
+
+```powershell
+$myObject.psobject.properties.Name[2]
+$myObject.psobject.properts.Value[2]
+```
+
```

Comment:
This is not an appropriate example. You example returns the **State** property, not the Name property.